### PR TITLE
Update broken profile links - issue #23

### LINKED ISSUE #23: profile link broken
**Describe the bug**
Clicking on User profile jaywahr link on README.md takes to GitHub 404 error page

**To Reproduce**
1. Go to 'README.md'
2. Scroll down to 'the-community' section
3. Click on 'jaywahr'

**Expected behavior**
GitHub user's profile page should display.

### PR DESCRIPTION
**Pull request by onFilm**
## Title
fix for issue #23 

This pull request (PR) is made in reference to:
issue #23 

# Checklist:

- [x] My code follows the style guidelines of this project
- [x] I have performed a self-review of my own code
- [ ] I have commented my code, particularly in hard-to-understand areas
- [ ] I have made corresponding changes to the documentation
- [ ] My changes generate no new warnings
- [ ] I have added tests that prove my fix is effective or that my feature works
- [ ] New and existing unit tests pass locally with my changes
- [ ] Any dependent changes have been merged and published in downstream modules

## Testing

- [ ] My submission passes the tests?
  - If you did additional testing, please describe in detail how you tested your changes
- [ ] I added new tests for core changes, as applicable?

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -103,10 +103,10 @@ Meet our [community](https://github.com/legesher/legesher/tree/master/community)
...
     <td align="center"><a href="https://github.com/takashiw"><img src="https://avatars1.githubusercontent.com/u/8411001?v=4" width="100px;" alt="Takashi Wickes"/><br /><sub><b>Takashi Wickes</b></sub></a><br /><a href="#design-takashiw" title="Design">🎨</a></td>
-    <td align="center"><a href="https://github.com/imgovind"><img src="https://avatars0.githubusercontent.com/u/4269318?v=4" width="100px;" alt="Govindarajan Panneerselvam"/><br /><sub><b>Govindarajan Panneerselvam</b></sub></a><br /><a href="https://github.com/madiedgar/legesher-pride/commits?author=imgovind" title="Documentation">📖</a></td>
+    <td align="center"><a href="https://github.com/imgovind"><img src="https://avatars0.githubusercontent.com/u/4269318?v=4" width="100px;" alt="Govindarajan Panneerselvam"/><br /><sub><b>Govindarajan Panneerselvam</b></sub></a><br /><a href="https://github.com/legesher/legesher-pride/commits?author=imgovind" title="Documentation">📖</a></td>
```
Comment (madiedgar) [inline README.md:106]: That's a great catch - thank you so much for finding this! 👍

File: README.md [hunk 2]
```diff
@@ -103,10 +103,10 @@ Meet our [community](https://github.com/legesher/legesher/tree/master/community)
...
     <td align="center"><a href="https://github.com/thiagondl"><img src="https://avatars0.githubusercontent.com/u/44266113?v=4" width="100px;" alt="Thiago Lima"/><br /><sub><b>Thiago Lima</b></sub></a><br /><a href="#ideas-thiagondl" title="Ideas, Planning, & Feedback">🤔</a></td>
     <td align="center"><a href="https://github.com/iMeriem"><img src="https://avatars1.githubusercontent.com/u/11720929?v=4" width="100px;" alt="Meriem Zaid"/><br /><sub><b>Meriem Zaid</b></sub></a><br /><a href="#ideas-iMeriem" title="Ideas, Planning, & Feedback">🤔</a></td>
-    <td align="center"><a href="https://github.com/jaywahr"><img src="https://avatars2.githubusercontent.com/u/6069258?v=4" width="100px;" alt="jaywahr"/><br /><sub><b>jaywahr</b></sub></a><br /><a href="https://github.com/madiedgar/legesher-pride/commits?author=jaywahr" title="Documentation">📖</a></td>
+    <td align="center"><a href="https://github.com/jaywar"><img src="https://avatars2.githubusercontent.com/u/6069258?v=4" width="100px;" alt="jaywar"/><br /><sub><b>jaywar</b></sub></a><br /><a href="https://github.com/legesher/legesher-pride/commits?author=jaywar" title="Documentation">📖</a></td>
```
Comment (madiedgar) [inline README.md:109]: Thank you for updating this 👍

**Review by madiedgar — APPROVED** [review]
Thanks for your contribution! 👍

**Comment by onFilm** [comment]
Thanks for approving the change, Can you please merge to master 😊

**Comment by madiedgar** [comment]
thanks for your patience with the merge.  When I added you as a contributor, I found why some of the links were updated incorrectly. Merging now! 👍

**Pull request merged**